intg/src/main/java/org/apache/atlas/ApplicationProperties.java

```diff
@@ -70,6 +70,7 @@ public final class ApplicationProperties extends PropertiesConfiguration {
     public static final boolean DEFAULT_INDEX_RECOVERY          = true;
     public static final AtlasRunMode DEFAULT_ATLAS_RUN_MODE     = AtlasRunMode.PROD;
     public static final String INDEX_SEARCH_MAX_RESULT_SET_SIZE = "atlas.graph.index.search.max-result-set-size";
+    public static final String DELTA_BASED_REFRESH              = "atlas.authorizer.enable.delta_based_refresh";
```

Comment:
Remove this from here, keep configuration managements in AtlasConfiguration.java